doc/sphinx-guides/source/developers/dependencies.rst

```diff
@@ -195,7 +199,7 @@ However, as life is a box of chocolates, you might find yourself in *version con
 knowing, but experiencing unintended side effects.
 
 When you look at the topmost graph above, imagine *B* and *TB* rely on different *versions* of *TC*. How does Maven
-decide which version it will include? Easy: the dependent version of the nearest version wins:
+decide which version it will include? Easy: the "nearest" dependency version wins. The following graph gives an example:
```

Comment:
This now better @pdurbin ?

Reply:
@poikilotherm it's better but nearest to what? Nearest to "Your Code"?

Reply:
I don't think we need to move the explanation.

I do think we should add "Your Code" or something in there. This explanation from https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Transitive_Dependencies is clear enough to me:

> it uses the version of the closest dependency to your project in the tree of dependencies

Reply:
I made a change in 8b30406. I hope it's ok with you.